sdk/spring/CHANGELOG.md

```diff
@@ -1,30 +1,109 @@
 # Release History
 
-## 4.0.0-beta.3 (Unreleased)
-- Supported spring-boot version: 2.6.0 - 2.6.1
-- Supported spring-cloud version: 2021.0.0
+## 4.0.0-beta.3 (2022-01-18)
+Please refer to [Spring Cloud Azure Migration Guide for 4.0](https://microsoft.github.io/spring-cloud-azure/4.0.0-beta.3/4.0.0-beta.3/reference/html/appendix.html#migration-guide-for-4-0) to learn how to migrate to version 4.0.
```

Comment:
why there are two `4.0.0-beta.3` here? is it reasonable?

Reply:
the deploy script will use the branch/version as the push path

Reply:
cool